### PR TITLE
#86 / changelog page

### LINKED ISSUE #86: Add a changelog page
**Is your feature request related to a problem? Please describe.**
We don`t have a changelog page

**Describe the solution you'd like**
Add a changelog page and use the changelog.json file from the game

### PR DESCRIPTION
**Pull request by ClientCrash**
**Description**
Adding a changelog Page

**Issue**
Closes #86 

**Testing Instructions**
Build the site. Everything works fine.

### CODE REVIEW

**Comment by ClientCrash** [comment]
@MarcelCoding @DevMimas @JannikEmmerich

File: pages/changelog.vue [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    NavigationBar
+  },
+  data () {
+    return {
+      changes: []
+    }
+  },
+  async fetch () {
+    const response = await fetch('https://play.cryptic-game.net/assets/changelog.json')
+    const data = await response.json()
```
Comment (Tert0) [inline pages/changelog.vue:59]:
```suggestion
    this.changes = await response.json()
```

File: pages/changelog.vue [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    this.changes = data
```
Comment (Tert0) [inline pages/changelog.vue:60]: And then delete this

**Review by Tert0 — APPROVED** [review]

**Review by JannikEmmerich — DISMISSED** [review]
Please check your code style (Actions fail)

**Comment by ClientCrash** [comment]
> Please check your code style (Actions fail)

I forgot to lint the code. I sorry.  @JannikEmmerich

**Comment by MarcelCoding** [comment]
Screen Shot 2021-08-30 at 22 04 20

File: pages/changelog.vue [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    <NavigationBar title="Changelog" />
+    <div class="content formatted">
+      <h1>{{ $t("navbar.changelog") }}</h1>
+
+      <div class="grid">
+        <div v-for="change in changes.versions" :key="change.name" class="versionEntry">
+          <article>
+            <div>
+              <h3>{{ change.name }}</h3>
+              <small>{{ change.date }}</small>
```
Comment (MarcelCoding) [inline pages/changelog.vue:12]: Please localize the date.
Maybe use something like this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

**Comment by ClientCrash** [comment]
nwueovbowuenvow

File: pages/changelog.vue (hunk 3 above)
Comment (ClientCrash) [reply pages/changelog.vue:12]: I will work on it if i find time.

**Comment by ClientCrash** [comment]
ewiubvizewgvizdsuhiu
New Design.

**Comment by ClientCrash** [comment]
IMAGE

**Comment by MarcelCoding** [comment]
`git apply <filename>`
[diff.txt](https://github.com/cryptic-game/website/files/7122038/diff.txt)

**Review by MarcelCoding — APPROVED** [review]

**Review by DevMimas — APPROVED** [review]

**Pull request merged**